packages/core/tests/ledger.test.ts

```diff
@@ -145,6 +145,7 @@ describe('ledger', () => {
       indyLedgers: [
         {
           id: 'pool-Faber Ledger Genesis Transactions',
+          didIndyNamespace: 'pool-Faber Ledger Genesis Transactions',
```

Comment:
I'm not sure namespaces can contain spaces?

Reply:
As per the did-core spec [here](https://www.w3.org/TR/did-core/#did-syntax)

Its defined like so (IMO not the most understandle way to define it...): 

```
did                = "did:" method-name ":" method-specific-id
method-name        = 1*method-char
method-char        = %x61-7A / DIGIT
method-specific-id = *( *idchar ":" ) 1*idchar
idchar             = ALPHA / DIGIT / "." / "-" / "_" / pct-encoded
pct-encoded        = "%" HEXDIG HEXDIG
```

The `didIndyNamespace` would be in the `method-specific-id` which does not allow spaces.

tl;dr no they can not.

Reply:
I think I have the same point as @TimoGlastra mentioned here: https://github.com/hyperledger/aries-framework-javascript/pull/965#issuecomment-1195815884